libs/openFrameworks/app/ofMainLoop.cpp

```diff
@@ -23,8 +23,6 @@
 	#include "ofAppGLFWWindow.h"
 #elif defined(TARGET_RASPBERRY_PI)
 	#include "ofAppEGLWindow.h"
-	#include "ofAppGLFWWindow.h"
```

Comment:
@dimitre - is this safe to remove?
I know a lot of people use `ofAppGLFWWindow.h` on Pi - or does it get included elsewhere ?

Reply:
Yes safe. this commit in fact reverts ofMainLoop.cpp to original state. I added this one in previous commits and removed now